pkg/webview/view.proto

```diff
@@ -119,7 +124,7 @@ message Resource {
   repeated string pending_build_edits = 9;
   google.protobuf.Timestamp pending_build_since = 10;
   bool has_pending_changes = 11;
-  repeated string endpoints = 12;
+  repeated Link endpoints = 12;
```

Comment:
as a general rule of thumb, never, ever change the type of a protobuf field. treat it like a database schema

in this particular case, this is persisted to a TFT database, so this change would break all existing snapshots. but the reason we have that general rule is so that there's a common understanding of what changes are OK, and you don't have to think too hard about versioning

instead, create a new field with the type you want

Reply:
whoooop yeah I was just seeing that this broke snapshots. will redo!

Reply:
is there a convention for how to name the new field vis a vis the old one, or anything?

Reply:
i would probably call it `links` (after the type) or `endpoints2` (to communicate that it replaces `endpoints`)